File: categorias/iniciante/c++/1043.cpp
#include <iostream>

using namespace std;

int main() {
    double n1, n2, n3;
    cout.precision(1);
    cout.setf(ios::fixed);

    cin >> n1 >> n2 >> n3;
    if (
        (n1 + n2) > n3 &&
        (n2 + n3) > n1 &&
        (n3 + n1) > n2
    ) {
        cout << "Perimetro = " << n1 + n2 + n3 << "\n";
    } else {
        cout << "Area = " << ((n1+n2)*n3)/2 << "\n";
    }

    return 0;
}

File: categorias/iniciante/python/1043.py
# -*- coding: utf-8 -*-

n1, n2, n3 = map(float, input().split())

if (n1+n2>n3 and n2+n3>n1 and n1+n3>n2):
    print('Perimetro =', n1+n2+n3)
else:
    print('Area =', (n1+n2)/2*n3)
